src/rpc/methods.zig

```diff
@@ -14,6 +19,329 @@ const Pubkey = sig.core.Pubkey;
 const Signature = sig.core.Signature;
 const Slot = sig.core.Slot;
 
+/// NOTE: for the sake of simplicity, we only support `method: ..., params: ...`,
+/// and reject `params: ..., method: ...`; this is a reasonable expectation for
+/// clients to satisfy.
+pub const Call = struct {
```

Comment:
I was confused by this name at first and had to read the implementation to understand that it represents requests. I think `Request` would be a clearer name.

Also I would move this to another file. I think it's cleaner if `methods.zig` is specialized to solely define the individual methods. Note that the `Response` struct is defined in its own file. I would probably move this struct into another file like `request.zig`

Reply:
669712e85df373d108f334f09681793ec459b267